### DISCUSSION TITLE
Is there a way to display charts in custom pages?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2731] [Help] [answered]
Hi guys, how are you?

Is there any way to display charts in custom pages?

Just started with them...

Thanks!

**Comment by danharrin** [comment, marked as answer]
Yes, chart widgets are Livewire components and you can render them as normal in custom views, using `@livewire`

**Reply by otavio-araujo** [reply]
Thank You Dan!
